lib/routes/one/namespace.ts

```diff
@@ -0,0 +1,7 @@
+import type { Namespace } from '@/types';
```

Comment:
Please use [namespace](https://docs.rsshub.app/joinus/new-rss/start-code#writing-radar-rules) `wufazhuce` instead of `one`.

Reply:
"wufazhuce" in Chinese means "unable to register." It is quite different from the website itself, even so, do we still have to use this second-level domain name?